x/capability/types/codec.go

```diff
@@ -20,9 +16,24 @@ func RegisterCodec(cdc *codec.Codec) {
 // RegisterCapabilityTypeCodec registers an external concrete Capability type
 // defined in another module for the internal ModuleCdc.
 func RegisterCapabilityTypeCodec(o interface{}, name string) {
```

Comment:
I don't think we need this anymore. Caps are never serialized.

Reply:
if we don't need to register capabilities, do we need the interface at all?

Reply:
We technically don't.